Neos.Neos/Classes/Controller/Module/Management/WorkspacesController.php

```diff
@@ -510,11 +511,17 @@ protected function computeSiteChanges(Workspace $selectedWorkspace)
                     // $document will be null if we have a broken root line for this node. This actually should never happen, but currently can in some scenarios.
                     if ($document !== null) {
                         $documentPath = implode('/', array_slice(explode('/', $document->getPath()), 3));
+                        $documentDimension = '';
+                        foreach ($document->getDimensions() as $dimension) {
```

Comment:
that seems dangerous? I would probably use the dimensionhash?

Reply:
dimensionhash is not available in postgres DBs yet afaik

Reply:
Must be, if not, that would be a grave bug that should have been noticed.

Reply:
I forked neos-development-collection and wrote a migration for it now: https://github.com/neos/neos-development-collection/pull/4429

Reply:
@Jonathan2022Bausch Which versions are you using? Tried it and I have the hash also with Postgres
![Screenshot 2023-08-17 at 14 22 39](https://github.com/neos/neos-development-collection/assets/1014126/37a0c524-5d30-4a9c-936c-d2ba867dd886)

Reply:
Ok, understood now the case with the PR https://github.com/neos/neos-development-collection/pull/4429